crates/type/src/repository/config.rs

```diff
@@ -5,6 +5,7 @@ use serde::{Deserialize, Serialize};
 
 /// A repository config
 #[derive(Clone, Debug, Default, Deserialize, Eq, PartialEq, Serialize)]
+#[allow(missing_copy_implementations)]
```

Comment:
Why?

Reply:
Because we're going to add more fields, which are most probably not going to be `Copy` and removing `Copy` from this struct would be a breaking change